Documentation/config-drive.md

```diff
@@ -2,10 +2,10 @@
 
 ---
 
-**NOTE**: coreos-cloudinit is no longer under active development and has been superseded by [Ignition][ignition]. For more information about the recommended tools for provisioning Container Linux, refer to the [provisioning documentation][provisioning].
+**NOTE**: This project overlaps in purpose with [Ignition][ignition] which is where most active development is taking place. However, the Flatcar Container Linux team also continues to support and maintain this project to maintain compatibility with cloudinit based environments.
```

Comment:
The "This project" part here is a bit confusing, given that this page is about Config Drive, not cloudinit. I'm not sure what the relationship between config drive and cloud init is, but I think here it should say "cloud init" rather than "This project". Or maybe just get rid of the whole note here. I really don't see how it's related.

Reply:
Copy-paste error, your suggestion makes sense. Will update.